recipe/meta.yaml

```diff
@@ -69,3 +83,4 @@ extra:
     - hanwen-pcluste
     - gmarciani
     - francesco-giordano
+    - ocefpaf
```

Comment:
can you remove your name from the maintainers?

AWS ParallelCluster is an AWS supported tool so we prefer to have full control of the released versions. Thanks!

Reply:
Sure but I am a core member of conda-forge and I already have write commits here. I only added myself so I remember to update this one b/c it was quite abandoned, many update PRs closed.

Reply:
I wasn't aware you were part of conda-forge, so thanks for working on this!

Reply:
BTW, we, core members, usually invite contributors who are picking up abandoned feedstocks. While having a single team taking care of it is nice, we do have many options in place to invite contribution without causing any harm. From reviews to marking packages as broken, reverting changes on GH, and more. TL;DR we rather have more help than leaving a package stale in the feedstock.

Reply:
Thank you for the detailed explanation!